frontend/packages/console-shared/src/components/editor/ShortcutsLink.tsx

```diff
@@ -15,9 +15,18 @@ const ShortcutsLink: React.FC<ShortcutsLinkProps> = ({ onHideShortcuts }) => {
       aria-label={t('editor~Shortcuts')}
       bodyContent={
         <ShortcutTable>
+          <Shortcut alt keyName="F1">
+            {t('editor~Accessibility help')}
+          </Shortcut>
+          <Shortcut keyName="F1">{t('editor~View all editor shortcuts')}</Shortcut>
           <Shortcut ctrl keyName="space">
             {t('editor~Activate auto complete')}
           </Shortcut>
+          <Shortcut ctrl toggleTabAction keyName="m">
```

Comment:
Remove custom prop `toggleTabAction`:
```suggestion
          <Shortcut ctrl shift={isMac} keyName="m">
```

Reply:
updated